merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/MerlinParsers.java

```diff
@@ -89,24 +89,25 @@ public JsonValue unparse(final Timestamp value) {
           untuple((role, userId) -> new HasuraAction.Session(role, userId.orElse(""))),
           $ -> tuple($.hasuraRole(), Optional.ofNullable($.hasuraUserId()))));
 
-  private static <S> JsonParser<Pair<Pair<String, S>, HasuraAction.Session>> hasuraActionP(final JsonParser<S> inputP) {
+  private static <I> JsonParser<Pair<Pair<Pair<String, I>, HasuraAction.Session>, Optional<String>>> hasuraActionP(final JsonParser<I> inputP) {
     return productP
-        .  field("action", productP.field("name", stringP))
-        .  field("input", inputP)
-        .  field("session_variables", hasuraActionSessionP);
+        .field("action", productP.field("name", stringP))
+        .field("input", inputP)
+        .field("session_variables", hasuraActionSessionP)
+        .optionalField("request_query", stringP);
```

Comment:
🔶 A "discussion question" of sorts is - is `request_query` truly optional, or does `v2.2` always provide it?

The optional seems like a reasonable way to support both `v2.0.10` and `v2.2`, but I just want to make sure I understand the motivation.

Reply:
Just pushed a change to make this a `field()` :)

Reply:
Also retested with this change. Looking good

Reply:
Looks fine to me - as long as we don't need to support older versions of hasura (I'm not the right person to make that decision)